Formula/links.rb

```diff
@@ -20,22 +21,16 @@ class Links < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "librsvg"
   depends_on "libtiff"
```

Comment:
I think these are useless without enabling graphics mode. Default is text mode.

Reply:
It's not enabled by default?

Reply:
Need to set `--enable-graphics`. Looks like we dropped support when options were removed: https://github.com/Homebrew/homebrew-core/commit/a8d7deb19cb9576c5303b460bf8fee666804f1de.

Reply:
Also needs x11, it seems. We can drop them.